app/src/main/java/org/glucosio/android/tools/GlucosioNotificationManager.java

```diff
@@ -68,4 +71,9 @@ public void sendReminderNotification() {
         NotificationManagerCompat notificationManagerCompat = NotificationManagerCompat.from(context);
         notificationManagerCompat.notify(NOTIFICATION_ID, notification);
     }
+
+    private int generateRandomNumber(int min, int max) {
+        Random r = new Random();
```

Comment:
I would say `Random` is extra here. You can just use `System.currentMillesecs() % 2`